### PR TITLE
fix translation german

### PR DESCRIPTION
**Pull request by idnovic**
I fixed the german translation as I saw fit.
It is around 95% good to go.

### CODE REVIEW

**Comment by idnovic** [comment]
I checked the new iOS build (1.7) and these are the changes.

File: src/locals/de.json [hunk 1]
```diff
@@ -59,18 +59,18 @@
     "bad_distortions": {
       "line1": "Bist du dir sicher George hast dich? Wenn nicht, bist du am",
```
Comment (cfleschhut) [inline src/locals/de.json]: @idnovic Minor error spotted here in this onboarding slide:
```diff
- Bist du dir sicher George hast dich?
+ Bist du dir sicher George hasst dich?
```

File: src/locals/de.json [hunk 2]
```diff
@@ -111,26 +111,26 @@
...
   "should_statements": "Sollte Aussagen",
-  "should_statements_explanation_1": "Wenn du jemanden nicht vorhandene Fähigkeiten zusprichst, benutzt du falsche \"sollte\" Aussagen. Als Beispiel, wenn du Flugangst hast und dir selbst einredest \"Ich sollte nicht Angst haben, es ist nichts falsch mir dem Flugzeug!\" legst du eine unnötige Last auf dich. Du hast Flugangst! Es ist normal für Menschen welche Angst vor dem Fliegen haben Angst beim Fliegen zu haben!",
-  "should_statements_explanation_2": "Sollte Aussagen klingen manchmal unsinnig, wenn du diese Laut aussprichst; das ist der Punkt! Sie sind unlogisch!",
+  "should_statements_explanation_1": "Wenn du jemanden nicht vorhandene Fähigkeiten zusprichst, benutzt du falsche \"sollte\" Aussagen. Als Beispiel, wenn du Flugangst hast und dir selbst einredest \"Ich sollte nicht Angst haben, es ist nichts falsch mit dem Flugzeug!\" legst du eine unnötige Last auf dich. Du hast Flugangst! Es ist normal für Menschen, welche Angst vor dem Fliegen haben, Angst beim Fliegen zu haben!",
+  "should_statements_explanation_2": "Sollte Aussagen klingen manchmal unsinnig, wenn du diesen Laut aussprichst; das ist der Punkt! Sie sind unlogisch!",
```
Comment (cfleschhut) [inline src/locals/de.json]: @idnovic Should read, I think:
```diff
- …, wenn du diesen Laut aussprichst`
+ …, wenn du diese laut aussprichst`

// "When you say it (these statements) out loud"
```

Comment (idnovic) [reply src/locals/de.json]: yes you are right

**Comment by idnovic** [comment]
I will rebase the translation on the current master in the following days

**Comment by idnovic** [comment]
#137

**Comment by idnovic** [comment]
@cfleschhut I wanted to ask you to review the new pull request. #137 It is nearly the same. I changed based on your suggestion.

**Comment by cfleschhut** [comment]
Hey @idnovic, sorry for the late reply! I looked into your PR/merged translations, looks very good to me 👍 Thanks for making these changes

**Pull request closed without merging**